eland/dataframe.py

```diff
@@ -1446,6 +1446,39 @@ def keys(self) -> pd.Index:
         """
         return self.columns
 
+    def iterrows(self) -> Iterable[Tuple[Hashable, Series]]:
```

Comment:
This function should return `Iterable[Tuple[Union[str, Tuple[str, ...]], pd.Series]]`